influxdb3/write.go

```diff
@@ -59,7 +61,17 @@ func (c *Client) WritePoints(ctx context.Context, points ...*Point) error {
 //
 // Returns:
 //   - An error, if any.
+//
+// Deprecated: use WritePoints with WriteOption options.
```

Comment:
Here too maybe the message could be more specific.  e.g. "Deprecated: use WritePoints with variadic WriteOption options"

Reply:
The message is updated.